packages/backend/src/redirect.ts

```diff
@@ -0,0 +1,24 @@
+type RedirectAdapter = (url: string, response?: any) => void;
+
+type SingUpParams = { signUpUrl: string; url: string; response?: any };
+type SingInParams = { signInUrl: string; url: string; response?: any };
+
+export class RedirectHelper {
```

Comment:
🙃 I'd suggest using JS POJOs instead of TS classes, as they add unnecessary bloat.

cc: @nikosdouvlis

Reply:
The only reasoning behind class instead of POJOs is to provide an API that you can define once how to the redirect works (parameter in constructor) and use it either for signup or singin redirections.
By changing it to JS POJOs i would have to add another parameter to each function (`toSignUp`/ `toSignIn`) for the redirectAdapter. Are we okay with that?

Reply:
An alternative option is to leverage closures

```
function (options...) {
  // closure vars
  return {
    redirectToSignIn,
    redirectToSignUp
  }
}
```

Reply:
Refactored as suggested. Could you take another look?